build/policy/files.rego

```diff
@@ -48,6 +49,19 @@ deny["Logo must be a .png or .svg file"] {
 	not extension(filename) in logo_exts
 }
 
+deny["Logo name must must match integration"] {
```

Comment:
Must must

Reply:
Thanks! Fixed. I also sneaked in `opa fmt --fail` as part of the build steps.